parts/k8s/windowskubeletfunc.ps1

```diff
@@ -410,35 +413,35 @@ if (`$hnsNetwork)
     docker ps -q | foreach {docker rm `$_ -f}
 
     Write-Host "Cleaning up old HNS network found"
-    Remove-HnsNetwork `$hnsNetwork
-    # Kill all cni instances & stale data left by cni
-    # Cleanup all files related to cni
-    taskkill /IM azure-vnet.exe /f
-    taskkill /IM azure-vnet-ipam.exe /f
-    `$cnijson = [io.path]::Combine("$KubeDir", "azure-vnet-ipam.json")
-    if ((Test-Path `$cnijson))
-    {
-        Remove-Item `$cnijson
-    }
-    `$cnilock = [io.path]::Combine("$KubeDir", "azure-vnet-ipam.json.lock")
-    if ((Test-Path `$cnilock))
-    {
-        Remove-Item `$cnilock
-    }
-
-    `$cnijson = [io.path]::Combine("$KubeDir", "azure-vnet.json")
-    if ((Test-Path `$cnijson))
-    {
-        Remove-Item `$cnijson
-    }
-    `$cnilock = [io.path]::Combine("$KubeDir", "azure-vnet.json.lock")
-    if ((Test-Path `$cnilock))
-    {
-        Remove-Item `$cnilock
-    }
-}
-
-# Restart Kubeproxy, which would wait, until the network is created
+    Remove-HnsNetwork `$hnsNetwork

```

Comment:
I do not see what changed here. Make sure you did not change EOL

Reply:
My ide must have changed these. The file is half CRLF and half LF. Which should I be using?

Reply:
Probably better to leave it as it was before. You or someone else can push a subsequent clean up PR.